ionos_sbx/issuer/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: issuer
```

Comment:
Please prefix the commit title with the environment name 'SBX'

Reply:
I added the enviroment prefix to the commit title